wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -437,6 +437,18 @@ definitions:
         - inbound
         - internal
         - outbound
+      remote_party:
+        type: object
+        properties:
+          kind:
```

Comment:
For homogeneity's sake, please keep `type` as keyword.

Reply:
Done